### ISSUE TITLE
Wrap component results into JsonObject

### ISSUE THREAD

**Issue by stas-fomenko** [elasticio/sailor-nodejs#118]
Wrap all results that are non-JSONObject into `{"result": <message>}`. 

Currently, we implement this in components, but it makes sense to implement it once in the sailor, as there are already at least 2 components that have they wrapped results into JSONObject in theirs code: restapi, salesforce (Magic updates). 

This will keep stable runtime mapper processing of messages at the next steps.

**Comment by hannah-bulmer** [comment]
Are there any components that might emit data as an array? Would this be backwards compatible for them if they became wrapped in a `{result: array}` object?

**Comment by stas-fomenko** [comment]
>Are there any components that might emit data as an array?

No, currently component output should be as a JSON Object, as platform mapper has some issues with arrays, so we try to avoid arrays in output

>Would this be backwards compatible for them if they became wrapped in a {result: array} object?

I'm not sure, we briefly discussed it with @kirill-levitskiy, but this task must be discussed with the whole team
